CustomeDB/Node.h

```diff
@@ -1,20 +1,16 @@
 #pragma once
 #include <string>
-#include "item.h"
+#include "Employee.h"
 
 using namespace std;
 
 class Node {
 private:
-    Item data;
+    Employee emp;
 public:
-    Node(int id, string name, int price);
+	Node();
+    Node(string name, int id, double salary, string project, string joiningDate, string phone, string address);
```

Comment:
Why is there a paramterized node constructor?

Reply:
i just found hafaz has implemented it this way. in that essence, its okay. ill approve it